packages/node-plop/tests/load-assets-from-plopfile/load-assets-from-plopfile.spec.js

```diff
@@ -173,4 +173,15 @@ describe("load-assets-from-plopfile", function () {
     expect(plop.getPartialList().length).toBe(1);
     expect(plop.getPartialList()[0]).toBe("p1");
   });
+
+  test.todo("plop.load loaded plopfile inherits config", async function () {
+    const plop = await nodePlop("", {
+      force: true,
+    });
+
+    await plop.load(plopfilePath, null, {});
+
+    // not a working test
+    expect(plop.config.force).toBe(true);
+  });
```

Comment:
Instead of a TODO test, let's add an `e2e` test using Plop CLI :)

Reply:
> e2e test using Plop CLI

Is there a prior art for this? Can't find anything like that in the repo.